src/scripts/components/MultiAvatar.jsx

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class MultiAvatar extends React.Component {
```

Comment:
I don't see the prototype, but is this ever more than two avatars?

Reply:
here's the proto: https://projects.invisionapp.com/share/FRTWGLBBC6T#/screens/380165608

Should just be two, per product